src/forder.c

```diff
@@ -1328,7 +1348,8 @@ SEXP binary(SEXP x)
 {
   char buffer[69];
   int j;
-  if (!isReal(x)) error(_("x must be type 'double'"));
+  if (!isReal(x))
```

Comment:
This function `binary(` isn't used anywhere... just for debugging?

Reply:
Yes